tests/ToolBox/ParameterAnalyzerTest.php

```diff
@@ -163,4 +164,76 @@ public function detectParameterDefinitionNone(): void
 
         self::assertNull($actual);
     }
+
+    #[Test]
+    public function getParameterDescriptionWithoutDocBlock(): void
+    {
+        $targetMethod = self::createStub(\ReflectionMethod::class);
+        $targetMethod->method('getDocComment')->willReturn(false);
+
+        $methodToTest = new \ReflectionMethod(ParameterAnalyzer::class, 'getParameterDescription');
+
+        self::assertSame(
+            '',
+            $methodToTest->invoke(
+                $this->analyzer,
+                $targetMethod,
+                'myParam',
+            )
+        );
+    }
+
+    #[Test]
+    #[DataProvider('provideGetParameterDescriptionCases')]
+    public function getParameterDescriptionWithDocs(string $docComment, string $expectedResult): void
+    {
+        $targetMethod = self::createStub(\ReflectionMethod::class);
+        $targetMethod->method('getDocComment')->willReturn($docComment);
+
+        $methodToTest = new \ReflectionMethod(ParameterAnalyzer::class, 'getParameterDescription');
+
+        self::assertSame(
+            $expectedResult,
+            $methodToTest->invoke(
+                $this->analyzer,
+                $targetMethod,
+                'myParam',
+            )
+        );
+    }
+
+    public static function provideGetParameterDescriptionCases(): \Generator
+    {
+        yield 'empty doc block' => [
+            'docComment' => '',
+            'expectedResult' => '',
+        ];
+
+        yield 'single line doc block with description' => [
+            'docComment' => '/** @param string $myParam The description */',
+            'expectedResult' => 'The description',
+        ];
+
+        yield 'multi line doc block with description and other tags' => [
+            'docComment' => '/** @param string $myParam The description
```

Comment:
What about using php multilingual variable for readability here?

Reply:
Preeeetty sure he meant multiline not multilingual :D

Reply:
Yes, sorry 🙈 autocorrection...

Reply:
Hehe. Yeah. Always hard to teach the smartphone. I have changed the texts to a nowdoc format. This should improve the readability.